Libraries/Components/Pressable/Pressable.js

```diff
@@ -51,6 +51,7 @@ type Props = $ReadOnly<{|
   accessibilityState?: ?AccessibilityState,
   accessibilityValue?: ?AccessibilityValue,
   accessibilityViewIsModal?: ?boolean,
+  'aria-modal'?: boolean,
```

Comment:
should probably be `?boolean`

Reply:
yes , i have pushed down the changes @necolas